proj2/xkobyd00_xzelen24.sql

```diff
@@ -193,13 +190,42 @@ CONSTRAINT FKVstupenkaProjekcia FOREIGN KEY
     ON DELETE CASCADE
 );
 
-    --vstupenky po zmazani projekcie ostanu kv�li vr�teniu pe?az�    
+    --vstupenky po zmazani projekcie ostanu kvoli vrateniu penaz�    
          
 
 ----------------------------INSERTION OF THE DATA STARTS HERE-----
+INSERT INTO multikino (nameCin, mesto, ulica)
+VALUES ('LuxCinematic', 'Brno', 'Metodejova');
+
+INSERT INTO zamestnanec  (rodneCislo, meno, priezvisko, adresa, cisloUctu, hodinovaMzda, pozicia, multikinoPraca)
+VALUES ('980604/6825', 'Andrej', 'Kristofan', 'Matice Slovinskej 1225/2 Brno', 'SK8975000000000012345671', 3.50, 'pokladnik', 'LuxCinematic');
+
+INSERT INTO zanerFilmu  (nazov, popis)
+VALUES ('thriller', 'Zaner, ktory ma u divaka vyvola silne napatie a emocie. Pozostava z rychleho sledu udalosti a castej akcie');
+
+INSERT INTO film (nazovFilmu, rok, klucSlova, reziser, trvanie, krajinaPovodu, vekoveObmedzenie)
+VALUES ('Mission: Impossible', 2018, 'action, hollywood, tom cruise', 'Christopher McQuarrie', 14, 'USA', 16);
+
+INSERT INTO premietaciaSala (kapacita, projektor, kino)
+VALUES (272, 'Benq W11000H', 'LuxCinematic');
+
+INSERT INTO projekcia (titulky, jazyk, datum, d3, projekciaFilm, projekciaSala)
+VALUES ('CZ', 'anglicky', to_timestamp('2019-11-24:20:45:00', 'YYYY-MM-DD:HH24:MI:SS'), 'y', TODO, TODO);
```

Comment:
@xDaile Tu nemám ako vložiť hodnotu do projekciaFilm a projekciaSala. Očakávajú sa tam kľučové premenné, ktoré sú vygenerované až za behu že? Takže tam nemám čo vložiť. Oprav ma ak sa mýlim, alebo ak to zle chápem.